VK/VK/GroupNewsScreen/GroupTableViewController.swift

```diff
@@ -147,5 +183,67 @@ final class GroupNewsTableViewController: UITableViewController {
     private func setupScene() {
         configureTableView()
         fetchPosts()
+        setupRefreshControll()
+    }
+
+    private func setupRefreshControll() {
+        refreshControl = UIRefreshControl()
+        refreshControl?.attributedTitle = NSAttributedString(string: Constant.refreshControllDefaultTitle)
+        refreshControl?.tintColor = .blue
+        refreshControl?.addTarget(self, action: #selector(refreshNews), for: .valueChanged)
+    }
+
+    @objc private func refreshNews() {
+        guard storyboardId == Constant.feedScreenStoryboardIDName else { return }
+        refreshControl?.beginRefreshing()
+        let mostFreshDate = posts.first?.date ?? Int(Date().timeIntervalSince1970)
+        networkService.fetchNews(startTime: mostFreshDate + 1) { [weak self] response in
+            guard let self = self else { return }
+            self.refreshControl?.endRefreshing()
+            switch response {
+            case let .success(success):
+                guard !success.response.items.isEmpty else { return }
+                for post in success.response.items where post.type == .post {
+                    let feedPost = NewsPostItem(feed: post)
+                    self.posts.append(feedPost)
+                }
+                let indexSet = IndexSet(integersIn: 0 ..< self.posts.count)
+                DispatchQueue.main.async {
+                    self.tableView.insertSections(indexSet, with: .automatic)
+                }
+            case let .failure(failure):
+                print(failure)
+            }
+        }
+    }
+}
+
+// Предзагрузка контента ячейки.
+extension NewsTableViewController: UITableViewDataSourcePrefetching {
+    func tableView(_ tableView: UITableView, prefetchRowsAt indexPaths: [IndexPath]) {
+        guard
+            let maxSection = indexPaths.map(\.section).max(),
+            maxSection > posts.count - 3,
+            !isLoading
+        else { return }
+        networkService.fetchMoreNews(startFrom: nextFrom) { response in
```

Comment:
```suggestion
        networkService.fetchMoreNews(startFrom: nextFrom) { response in
```
аналогично, как и у меня. я бы вынес в отдельный метод)